home.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900&display=swap" rel="stylesheet">
+  <script src="https://kit.fontawesome.com/c3c0d932d2.js" crossorigin="anonymous"></script>
+  <title>Home</title>
+</head>
+<body>
+  <header class="navbar">
+    <i class="fa-solid fa-bars menu-btn" id="menu-btn"></i>
+    <div class="navbar-menu">
+      <div class="menu-options">
+        <ul class="options">
+          <li class="nav-icon"><i class="fa-brands fa-facebook"></i></li>
+          <li class="nav-icon"><i class="fa-brands fa-square-twitter"></i></li>
+          <li class="nav-option">English</li>
+          <li class="nav-option">My page</li>
+          <li class="nav-option">Logout</li>
+        </ul>
+      </div>
+      <nav class="menu-links" id="menu">
+        <i class="fa-regular fa-circle-xmark close-btn menu-btn" id="close-btn"></i>
+        <a href="/home.html"><img src='media/event-logo.png' alt='logo' class='nav-logo' id='nav-logo'></a>
+        <ul class="nav-links">
+          <li class="nav-link"><a href="about.html">About</a></li>
+          <li class="nav-link"><a href="#">Program</a></li>
+          <li class="nav-link"><a href="#">Join</a></li>
+          <li class="nav-link"><a href="#">Sponsor</a></li>
+          <li class="nav-link"><a href="#">News</a></li>
+          <li class="nav-link"><a href="#">YWCC Campaign</a></li>
+        </ul>
+      </nav>
+    </div>
+  </header>
+
+  <div class="home-page active" id="home-page">
+    <section class="home">
+      <h2 class="subtitle-home">"Hello! Sharing World"</h2>
+      <h1 class="p-title">YEARLY WORLD COMEDIANS CONFERENCE</h1>
+      <p class="description">A joyful reunion of the most popular living comedians from all over the globe, to celebrate comedy, taking place this December in Spain.</p>
+      <div class="date-and-place">
+        <p class="date">2022.12.15(THU) - 16(FRI)</p>
+        <p class="place">@ Camp Nou, C. d'Arístides Maillol, 12, 08028 Barcelona, Spain</p>
+      </div>
+    </section>
```

Comment:
It looks great, but it's highly recommended to follow the original design idea as much as possible. Especially the colors, typography, and sizing. As you can see in the header the text size is much smaller than the original one, and the color is also not similar to the one in the original design **(Idea: Using the CSS [gradients](https://developer.mozilla.org/en-US/docs/Web/CSS/gradient/linear-gradient) can be helpful)**. Please make sure to increase the text size and use a similar color.


Yours
![image](https://user-images.githubusercontent.com/34779928/203856563-f3a2fd05-d850-42d8-8d16-3dce19b52b10.png)

Original
![image](https://user-images.githubusercontent.com/34779928/203856678-a742e561-165c-49ea-9926-550f6cea622d.png)